serveradmin/common/templates/base.html

```diff
@@ -79,6 +79,7 @@
 <script src="{{ STATIC_URL }}js/js.cookie.min.js"></script>
 <script src="{{ STATIC_URL }}js/serveradmin.js"></script>
 <script src="{{ STATIC_URL }}js/plugins/terminal_keyboard.js"></script>
+<script src="{{ STATIC_URL }}js/plugins/esc_dismissible.js"></script>
```

Comment:
I wonder if it would make more sense to put it into the `servershell.js`.

Reply:
I first put it to the servershell app but then decided that it could be handy to be able to use this functionality in other apps as well. This allows us to just add a CSS class dismissible to any HTML element and discard it with escape.

Reply:
Fair enough! 👍🏼